yarn-project/acir-simulator/src/avm/opcodes/index.ts

```diff
@@ -1,5 +1,11 @@
 export * from './arithmetic.js';
+export * from './bitwise.js';
 export * from './control_flow.js';
 export * from './instruction.js';
 export * from './comparators.js';
 export * from './memory.js';
+export * from './storage.js';
+// FIXME: dependency cycle
+// export * from './external_calls.js';
+export * from './environment_getters.js';
```

Comment:
Can we create a ticket for this? We are accumulating a lot of TODOs/FIXMEs it's going to get hard to keep track.

Reply:
Done! https://github.com/AztecProtocol/aztec-packages/issues/4359